source/common/http/request_id_extension_uuid_impl.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "envoy/http/request_id_extension.h"
+
+#include "common/runtime/runtime_impl.h"
+
+namespace Envoy {
+namespace Http {
+
+class UUIDRequestIDExtension : public RequestIDExtension {
```

Comment:
nit: small comment above that this is the default implementation if none is configured?